src/ngx_http_lua_kong_log_handler.c

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2019-2023 Kong Inc.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *    http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ngx_http_lua_kong_common.h"
+
+
+static u_char *
+ngx_http_lua_kong_error_log_handler(ngx_http_request_t *r, u_char *buf, size_t len)
+{
+    ngx_int_t                    req_id_var_index;
+    ngx_http_variable_value_t    *value;
+    ngx_http_lua_kong_loc_conf_t *lcf;
+
+    lcf = ngx_http_get_module_loc_conf(r, ngx_http_lua_kong_module);
+    req_id_var_index = lcf->req_id_var_index;
+    if (req_id_var_index == NGX_CONF_UNSET) {
```

Comment:
`req_id_var_index` seems to be unnecessary, we can use `lcf->req_id_var_index` directly.